core/service.go

```diff
@@ -359,6 +349,7 @@ func (svc *Service) startContainer(
 
 			res, err := bk.Solve(ctx, bkgw.SolveRequest{
 				Definition: def,
+				Evaluate:   true,
```

Comment:
I think this means that we will be solving each of these mount inputs one-by-one in serial, which would invalidate the later resolving of them in parallel: https://github.com/sipsma/dagger/blob/dc83928c3a13d6b315bf0953befde001ec359238/engine/buildkit/client.go#L223-L223

Somewhat obscure, but solving in parallel feels like an optimization worth keeping here if at all possible. Not sure how load-bearing setting Evaluate to true was.